rllib/core/rl_trainer/trainer_runner.py

```diff
@@ -47,56 +52,68 @@ class TrainerRunner:
 
     def __init__(
         self,
-        trainer_class: Type[RLTrainer],
-        trainer_config: Mapping[str, Any],
-        compute_config: Mapping[str, Any],
+        rl_trainer_spec: RLTrainerSpec,
+        scaling_config: Optional[TrainerScalingConfig] = None,
     ):
-        num_gpus = compute_config.get("num_gpus", 0)
-        use_fake_gpus = compute_config.get("_use_fake_gpus", False)
-        self._trainer_config = trainer_config
-
-        if num_gpus > 0:
-            scaling_config = ScalingConfig(
-                num_workers=num_gpus,
-                use_gpu=(not use_fake_gpus),
+        scaling_config = scaling_config or TrainerScalingConfig()
+        rl_trainer_class = rl_trainer_spec.rl_trainer_class
+
+        # setup wether the worker should use gpu or not
+        if rl_trainer_class.framework == "torch":
+            trainer_should_use_gpu = scaling_config.num_gpus_per_worker > 0
+            rl_trainer_spec.module_backend_config.set_use_gpu(trainer_should_use_gpu)
+        else:
+            # TODO (Avnish) How do I run TF on one GPU?
+            pass
+
+        self._is_local = scaling_config.num_workers == 0
+        if self._is_local:
```

Comment:
what does local and remote mode mean here?
is it ray core's local mode?
or do you mean rank=0 trainer?

Reply:
It means that the trainer lives on the same process as the driver. I.e. It's not an actor.

Reply:
so instead of _local, maybe we should do ``self._remote``?